### PR TITLE
fix: token and test

### PR DESCRIPTION
**Pull request by Marine-Berthier**
[This PR](https://github.com/smg-automotive/components-pkg/pull/430) was merged too quickly.

Here I adjust it according to the comments I missed.

### CODE REVIEW

File: src/components/tab/__tests__/Tabs.Test.tsx [hunk 1]
```diff
@@ -46,17 +46,17 @@ describe('<Tabs />', () => {
     expect(screen.getByText('Eins')).toBeInTheDocument();
     expect(screen.queryByText('Zwei')).not.toBeInTheDocument();
   });
-  it('changes tab on click', async () => {
```
Comment (Marine-Berthier) [inline src/components/tab/__tests__/Tabs.Test.tsx:49]: I did some tests and forgot to remove it

File: src/themes/components/tabs.ts [hunk 2]
```diff
@@ -1,11 +1,14 @@
+import { opacity } from '../shared/opacity';
+import { borders } from '../shared/borders';
+
 const baseStyleTab = {
   textStyle: 'body',
   transitionProperty: 'common',
   transitionDuration: 'normal',
-  borderBottom: '1.5px solid',
+  borderBottom: borders['1px'],
```
Comment (Marine-Berthier) [inline src/themes/components/tabs.ts:8]: Sergiu confirmed 1px 👍

Comment (GutierrezGo) [reply src/themes/components/tabs.ts:8]: ❤️

File: src/components/tab/__tests__/Tabs.Test.tsx (hunk 1 above)
Comment (GutierrezGo) [reply src/components/tab/__tests__/Tabs.Test.tsx:49]: Thank you!

File: src/themes/components/tabs.ts (hunk 2 above)
Comment (GutierrezGo) [reply src/themes/components/tabs.ts:8]: For the borders you don't need to reference it with an object.

Chakra has the key for the borders but no for opacity. The reason why we need it only for opacity

**Review by GutierrezGo — APPROVED** [review]
Thank you so much for addressing my late comments 🌸

**Pull request merged**